native_client/ctcdecode/Makefile

```diff
@@ -17,12 +17,13 @@ endif
 
 all: bindings
 
-clean-keep-common:
+clean-keep-archive:
 	rm -rf dist temp_build ds_ctcdecoder.egg-info
 	rm -f swigwrapper_wrap.cpp swigwrapper.py
 
-clean: clean-keep-common
-	rm -f common.a
+clean: clean-keep-archive
```

Comment:
And then this one deletes the third party archive too (changes less often).

Reply:
sorry, tired and the rules are really confusing when you are constantly distracted